modules/checkout/templates/commerce-checkout-form.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+/**
+ * @file
+ * Two column template for the checkout form.
+ *
+ * Available variables:
+ * - form: The form.
+ *
+ * @ingroup themeable
+ */
+#}
+<div class="layout-checkout-form clearfix">
+  <div class="layout-region layout-region-checkout-main">
+    {{ form|without('cart_summary', 'actions') }}
+  </div>
+  <div class="layout-region layout-region-checkout-secondary">
+    <h3>{% trans %} Order Summary {% endtrans %}</h3>
```

Comment:
Can the Order Summary title not come from the form element that's returned by buildCartSummary()?